diff --git a/source/creator/CMakeLists.txt b/source/creator/CMakeLists.txt
index fb02a7c871c..177cae16f1a 100644
--- a/source/creator/CMakeLists.txt
+++ b/source/creator/CMakeLists.txt
@@ -564,7 +564,6 @@ if(UNIX AND NOT APPLE)
 				DESTINATION ${TARGETDIR_VER}/python/${_target_LIB}
 				PATTERN "__pycache__" EXCLUDE               # * any cache *
 				PATTERN "config-${PYTHON_VERSION}m/*.a" EXCLUDE  # static lib
-				PATTERN "lib2to3" EXCLUDE                   # ./lib2to3
 				PATTERN "site-packages/*" EXCLUDE           # ./site-packages/*
 				PATTERN "tkinter" EXCLUDE                   # ./tkinter
 				PATTERN "lib-dynload/_tkinter.*" EXCLUDE    # ./lib-dynload/_tkinter.co
@@ -865,7 +864,6 @@ elseif(APPLE)
 			PATTERN "__MACOSX" EXCLUDE
 			PATTERN ".DS_Store" EXCLUDE
 			PATTERN "config-${PYTHON_VERSION}m/*.a" EXCLUDE  # static lib
-			PATTERN "lib2to3" EXCLUDE                   # ./lib2to3
 			PATTERN "tkinter" EXCLUDE                   # ./tkinter
 			PATTERN "lib-dynload/_tkinter.*" EXCLUDE    # ./lib-dynload/_tkinter.co
 			PATTERN "idlelib" EXCLUDE                   # ./idlelib
